gallery/src/components/NxTransferListHalf/NxTransferListHalfPage.tsx

```diff
@@ -90,8 +99,8 @@ const NxTransferListPage = () =>
             <NxTable.Row>
               <NxTable.Cell><NxCode>isSelected</NxCode></NxTable.Cell>
               <NxTable.Cell>boolean</NxTable.Cell>
-              <NxTable.Cell>Yes</NxTable.Cell>
-              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>false</NxTable.Cell>
```

Comment:
IMO we should actually make this default to true.  In cases where you have just a single `NxTransferListHalf`, it would presumably be holding things that are selected.